filecoin-proofs/src/post_adapter.rs

```diff
@@ -66,7 +66,7 @@ pub fn generate_post_spread_input(
 
     for chunk in chunks {
         let mut input_parts: [(Option<String>, Commitment); POST_SECTORS_COUNT] =
-            Default::default();
+            array![(None, [0u8;32]); POST_SECTORS_COUNT];
```

Comment:
blocking: none of these should not be stack allocated arrays, please use a Vec

Reply:
512 is simply too large for many stacks, the inner ones can stay as arrays

Reply:
Note for future programmers: We did a little [messing around](https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&gist=985091298e3700c643a0768cfe23c560) with `mem::size_of_val`, and it looks like a value of type `[(Option<String>, Commitment); 512]` takes up 28672B of stack. So we should be g2g.